modules/internal/String.chpl

```diff
@@ -373,10 +471,18 @@ module String {
       return ret;
     }
 
+    /*
+      :returns: * `true`  -- when the string begins with one or more of the `needles`
+                * `false` -- otherwise
```

Comment:
Though it's implied, for the sake of clarity I would also specify what "needles" represents